mcc/GetSetAttrs.c

```diff
@@ -418,16 +419,12 @@ IPTR mSet(struct IClass *cl, Object *obj, struct opSet *msg)
               data->ypos = _mtop(obj) - ti_Data%data->fontheight;
               lines = (-smooth/data->fontheight)+2;
 
-              {
-                struct Layer *layer = data->rport->Layer;
-
-                if(layer->DamageList && layer->DamageList->RegionRectangle)
-                  if(MUI_BeginRefresh(muiRenderInfo(obj),0))
-                  {
-                    MUI_Redraw(obj, MADF_DRAWOBJECT);
-                    MUI_EndRefresh(muiRenderInfo(obj), 0);
-                  }
-              }
+              if(layer->DamageList && layer->DamageList->RegionRectangle)
+                if(MUI_BeginRefresh(muiRenderInfo(obj),0))
```

Comment:
No, in C programming all evaluations in an ìf()` are evaluated from left to right and as soon as one is not matching  (due to the fact you combine with `&&`) the others more far right are not evaluated/executed at all. That means, if you do e.g.

```
if(a && printf("blah")) { ... }
```

Then `printf("blah")` will never be executed and thus no output written to the console if a is already false/0.